javascript.js

```diff
@@ -0,0 +1,10 @@
+function alertDogInfo(name, fee, descript){
+  let result = name + fee
+  alert("Name: " + name + "\n" + "Adopt Fee: " + fee + "\n" +  "Breed: " + descript )
+}
+
+function addTotal(number){
```

Comment:
this doesn't quite do what it is supposed to do. There should be a running total, and every time you click a button, the total price goes up by the price of the dog.